test/built-ins/Object/prototype/isPrototypeOf/null-context-and-object-value-throws.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2011 the Sputnik authors. All rights reserved.
```

Comment:
Just another nit pick: avoid referencing `this` as `context`. They are not fully the same thing spec wise.

Ideally the file names would be like `null-this-and-object-arg-throws.js`. See this file name also specify the arg is the object.

Reply:
Thank you, I've renamed `context` to `this` and `value` to `arg` & rebased.